app/helpers/application_helper/page_layouts.rb

```diff
@@ -53,6 +53,7 @@ def layout_uses_listnav?
   end
 
   def layout_uses_paging?
+    return true if @layout.starts_with?("miq_request")
```

Comment:
Instead of this I'd probably just add `miq_request_ae miq_request_host miq_request_vm` to the list
below. Not a big deal, it just looks nicer.